docs/tutorials/q32d14.ipynb

```diff
@@ -0,0 +1,3360 @@
+      "(6, 2): ───X^0.5───H^0.5─────────X^-0.5───────────Rz(-0.408π)───iSwap──────────@──────────Rz(0.471π)────Y^0.5────────────Rz(1.46π)────iSwap^-0.949───@^-0.145──────Rz(0.262π)──────X^0.5─────────────────────H^0.5───────────────────X^-0.5─────────────────────Rz(-0.982π)─────────────iSwap────────────────────────────────────────────────────────@────────────────────────────────────────────────────────────────Rz(-1.63π)──────────────X^0.5─────────Rz(-1.61π)─────iSwap^-0.971┼───────────┼───────────┼───────────────────@^(-2/13)──┼───────┼───────┼─────────────────────Rz(-1.97π)───────────────────────────────Y^0.5───────────────────────────Rz(-1.29π)───────────────────────────────────iSwap──────────@──────────────Rz(1.36π)─────X^0.5────────────Rz(-1.79π)────iSwap^-0.949────@^-0.145───────────────────Rz(-0.482π)─────────────Y^0.5────────Rz(1.08π)─────────────────iSwap────────────────────────────@───────────────────────Rz(0.307π)───────────────────────────────────────X^0.5───────────────────────────────────────────H^0.5────────────────────────────────────────X^-0.5─────────────────────
```

Comment:
Can you add some explanation here? What are we expecting to see and why?

Reply:
Added.